modules/ipi-install-bmc-addressing.adoc

```diff
@@ -77,7 +111,27 @@ platform:
           password: <password>
 ----
 
+
+.RedFish Virtual Media for Dell
+
+For RedFish virtual media on Dell servers, use `idrac-virtualmedia://` in the `address` setting. The following example demonstrates using iDRAC Virtual Media within the  `install-config.yaml` file.
```

Comment:
Is virtual media a proper noun? If so, the first instance needs capitalized here